### PR TITLE
Adding Installation Review/Progress/Summary pages

### PR DESCRIPTION
**Pull request by blrm**

### CODE REVIEW

File: pages/wizard/review/installation_progress.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    _details_tab_loc = (By.XPATH, '//li[contains(@class,"ember-view")]/a[text()="Details"]')
+    _log_tab_loc = (By.XPATH, '//li[contains(@class,"ember-view")]/a[text()="Log"]')
+
+    _error_checkbox_loc = (By.XPATH, '//input[@name="error"]')
+    _warn_checkbox_loc = (By.XPATH, '//input[@name="warn"]')
+    _info_checkbox_loc = (By.XPATH, '//input[@name="info"]')
+    _debug_checkbox_loc = (By.XPATH, '//input[@name="debug"]')
+    _logs_dropdown_loc = (By.XPATH, '//select[@id="log-file-select"]')
+
+    # log_select.select_by_value('foreman_log')
```
Comment (jamesooden) [inline pages/wizard/review/installation_progress.py]: I would just delete the commented out code.

File: pages/wizard/review/installation_review.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    def rhev_admin_pw_eye_icon(self):
+        return self.selenium.find_element(*self._rhev_admin_pw_eye_loc)
+
+    @property
+    def deploy_button(self):
+        return self.selenium.find_element(*self._deploy_button_loc)
+
+    # actions
+
+    def reveal_rhev_root_pw(self):
```
Comment (jamesooden) [inline pages/wizard/review/installation_review.py]: Aren't there others like this?

**Comment by jamesooden** [comment]
:+1: 

I approved through the other mechanism, but we haven't decided to use it so I did it this way too.

**Comment by apagac** [comment]
:+1: 
With comments from James.

File: pages/wizard/review/installation_review.py (hunk 2 above)
Comment (blrm) [inline pages/wizard/review/installation_review.py]: Added reveal password for CFME admin, root, and database. There are others for OSP and OSE, but I'll need to do a deployment with those to get the locators for them to add in a separate PR.

File: pages/wizard/review/installation_progress.py (hunk 1 above)
Comment (blrm) [inline pages/wizard/review/installation_progress.py]: deleted

**Pull request merged**